pkg/util/helper/mcs.go

```diff
@@ -106,34 +106,34 @@ func MultiClusterServiceCrossClusterEnabled(mcs *networkingv1alpha1.MultiCluster
 	return false
 }
 
-func GetProviderClusters(client client.Client, mcs *networkingv1alpha1.MultiClusterService) (existClusters, noneExistClusters sets.Set[string], err error) {
+func GetProviderClusters(client client.Client, mcs *networkingv1alpha1.MultiClusterService) (sets.Set[string], error) {
```

Comment:
Please help add some comments for this function. Same with other export functions.

I remember that our CI had requirements for comments on the export method, but I'm not sure why it is no longer effective.

Reply:
> I remember that our CI had requirements for comments on the export method, but I'm not sure why it is no longer effective.

I'm curious about it too.

Reply:
Added